samples/apps/copilot-chat-app/scripts/Start.ps1

```diff
@@ -1,44 +1,16 @@
 <#
 .SYNOPSIS
-Initializes and runs both the backend and frontend for Copilot Chat.
-
-.PARAMETER ClientId
-The client (application) ID associated with your AAD app registration.
-
-.PARAMETER Tenant
-The tenant associated with your AAD app registration.
-Defaults to 'common'.
-
-.PARAMETER AzureOpenAIOrOpenAIKey
-Your Azure OpenAI or OpenAI API key.
+Builds and runs both the backend and frontend for Copilot Chat.
 #>
 
 #Requires -Version 6
```

Comment:
Not sure if this is still needed here (looks like it's been removed in Start-Frontend and Start-Backend). Might only be necessary for the configure script? Or do we want to keep it in all for consistency?